src/util/BitField.js

```diff
@@ -48,16 +48,15 @@ class BitField {
   /**
    * Gets all given bits that are missing from the bitfield.
    * @param {BitFieldResolvable} bits Bit(s) to check for
-   * @param {...*} hasParams Additional parameters for the has method, if any
    * @returns {string[]}
    */
-  missing(bits, ...hasParams) {
-    return new this.constructor(bits).remove(this).toArray(...hasParams);
```

Comment:
This cannot be removed, its for inherited classes like Permissions that do have additional parameters in .has

Reply:
I cannot find any class that overwrites the has method and adds other parameters, could you link me one?

Reply:
<https://discord.js.org/#/docs/main/master/class/Permissions?scrollTo=has>